web/modeshape-web-explorer/pom.xml

```diff
@@ -28,7 +28,6 @@
         <dependency>
             <groupId>org.modeshape</groupId>
             <artifactId>modeshape-jcr-api</artifactId>
-            <scope>provided</scope>
```

Comment:
i'm curios - why the need for this change ? How is this dependency different than `javax.jcr` for example, which is still `provided` ?